docs/Internal Systems/Gatsby_Starter_Guide.md

```diff
@@ -0,0 +1,28 @@
+# Gatsby Starter Guide
+Gastby is static site generator based on react. To boost performance, GatsbyJS loads only critical HTML, CSS, JavaScript, and other data. Once loaded, it prefetches resources for other pages you might navigate to. This way, your site loads as fast as possible and user interactions feel seamless. 
+
+The main thing about our project and Gatsby is that we are using no database and no server side code. Now, to use gatsby you need understanding of ReactJS and JavaScript. The videos down below gives a good overview of react. 
+
+
+https://www.youtube.com/watch?v=hQAHSlTtcmY&&t=897s
+
+
+[https://youtu.be/hQAHSlTtcmY?t=897s](https://youtu.be/hQAHSlTtcmY?t=897s)
+
+Just understanding the basics of react should be enough to start using GatsbyJS. The main focus should be how the functionional components work. We will be creating our website in components as well and and then putting it all together.
```

Comment:
Perhaps put a link to React? Rather than saying "Just understanding the basics of react" perhaps change this to "Understanding the basic concepts of React is fundamental to start using Gatsby JS". Assume the people readying this have no experience